src/create-messaging.js

```diff
@@ -0,0 +1,47 @@
+import { Messaging, MessagingContext, WebkitMessagingConfig, WindowsMessagingConfig } from '@duckduckgo/messaging'
+
+/**
+ * Extracted so we can iterate on the best way to bring this to all platforms
+ * @param {import('./content-feature.js').default} feature
+ * @param {string} injectName
+ * @return {Messaging}
+ */
+export function createMessaging (feature, injectName) {
+    const contextName = injectName === 'apple-isolated'
+        ? 'contentScopeScriptsIsolated'
+        : 'contentScopeScripts'
+
+    const context = new MessagingContext({
```

Comment:
This will need to be refactored when Frans code lands, can you file a follow up issue to track?